addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5428,12 +5436,12 @@ msgstr ""
 
 #: xbmc\osx\XBMCHelper.cpp
 msgctxt "#13144"
-msgid "Change Apple Remote mode?"
+msgid "Change Apple remote mode?"
```

Comment:
No, remote is not a capitalized letter, this is exactly why people capitalize second words where they dont belong.